### DISCUSSION TITLE
GAST and how to use it

### DISCUSSION THREAD

**Discussion by bathoorn** [skyfielders/python-skyfield#942] [Q&A] [answered]
```
from datetime import datetime
from pytz import timezone
from skyfield.api import load
ts = load.timescale()
gmt = timezone('GMT')
d = datetime(2024, 3, 20, 3, 7, 0, tzinfo=gmt)
t = ts.from_datetime(d)
t.gast
=> 14.993056580644776
```

I am trying to figure out how to use the GAST value from a time object. And i think i am missing a step somewhere.
If i take as a datetime the vernal equinox in GMT timezone. Should then the t.gast value be somewhere close to zero?
As at GMT my longitude is 0 degrees. Then LST should be GAST - longitude. Or can i not use it in this way?

**Comment by brandon-rhodes** [comment, marked as answer]
The vernal equinox can happen at any time of day; it's not guaranteed to happen at noon or midnight. It's the moment when the Earth in its orbit is pointing its equator directly at the Sun, which is an event that ignores entirely the Earth's rotation.

**Reply by bathoorn** [reply]
But it happens once a year around the 20th of March right?

**Reply by brandon-rhodes** [reply]
Yes, once a year.

**Reply by bathoorn** [reply]
Ah i think is see the mistake in my reasoning

```
The Sun is at right ascension (RA) = 0h on the Vernal (Spring) Equinox in mid March, so in the spring the sky near RA = 0h will be visible during the day when the Sun is up.
```

So then the sun will be at 0h on the date and time of the vernal equinox.

thank for your time

**Reply by brandon-rhodes** [reply]
Yes! Zero hours right ascension, a coordinate in the sky; not zero hours in time, which would be midnight.
